css-borders-4/Overview.bs

```diff
@@ -474,28 +472,50 @@ Corner Shaping: the 'corner-shape' and 'corner-*-shape' properties</h4>
 <h4 id=corner-shape-interpolation>
 Interpolating corner shapes</h4>
 
-Since a <<corner-shape-value>> can always be expressed by a ''superellipse()'' with an [=superellipse exponent=] variable, interpolating between two
-<<corner-shape-value>>s is done by interpolating the [=superellipse exponent=] itself.
-Since it's an exponent, interpolating it linearly would result in an effect where concave corners interpolate at a much higher velocity than convex corners.
-To balance that, the <dfn>superellipse interpolation</dfn> formula describes how a [=superellipse exponent=] is converted to a value between 0 and 1, and vice versa:
-
-<div algorithm="superellipse-exponent-to-interpolation-value">
-To interpolate a <<number [0,&infin;]>> |exponent| to an interpolation value between 0 and 1:
-	1. If |exponent| is 0, return 0.
-	1. If |exponent| is &infin;, return 1.
-	1. Return <code>1/(2^(1/|exponent|))</code>.
-
-To convert a <<number [0,1]>> |interpolationValue| back to a [=superellipse exponent=]:
-	1. If |interpolationValue| is 0, return 0.
-	1. If |interpolationValue| is 1, return &infin;.
-	1. Return <code>ln(0.5)/ln(|interpolationValue|)</code>.
-</div>
+Since a <<corner-shape-value>> can always be expressed by a ''superellipse()'' with an [=superellipse parameter=] variable, interpolating between two
+<<corner-shape-value>>s is done by interpolating the [=superellipse parameter=] itself.
+Since it uses a <code>log2</code>, interpolating it linearly would result in an effect where concave corners interpolate at a much higher velocity than convex corners.
+To balance that, the <dfn>superellipse interpolation</dfn> formula describes how a [=superellipse parameter=] is converted to a value between 0 and 1, and vice versa:
 
-<a href="https://github.com/w3c/csswg-drafts/issues/11608">Issue #11608</a>: resolve on this or another interpolation formula.
+<div algorithm="superellipse-param-to-interpolation-value">
+To interpolate a <<number [-&infin;,&infin;]>> |param| to an interpolation value between 0 and 1, return the first matching statement,switch on |param|:
+<dl class=switch>
+	: -&infin;
+	:: Return 0.
 
-<pre>
+	: &infin;
+	:: Return 1.
 
-</pre>
+	: Otherwise
+	::
+		1. Let |convexHalfCorner| be <code>0.5<sup>0.5<sup>abs(|param|)</sup></sup></code>.
+		1. If |param| is less than 0, return <code>1 - |convexHalfCorner|</code>.
+		1. Return |convexHalfCorner|.
+
+</dl>
+
+To convert a <<number [0,1]>> |interpolationValue| back to a [=superellipse parameter=], switch on |interpolationValue|:
+<dl class=switch>
+	: 0
+	:: Return -&infin;.
+
+	: 0.5
+	:: Return 0.
+
+	: 1
+	:: Return &infin;.
+
+	: Otherwise
+	::
+		1. Let |convexHalfCorner| be |interpolationValue|.
+		1. If |interpolationValue| is less than 0.5, set |convexHalfCorner| to 1 - |interpolationValue|.
+		1. Let |exponent| be <code>log<sub>0.5</sub>(log<sub>0.5</sub>(|convexHalfCorner|))</code>.
```

Comment:
I computed this as `k = ln(0.5) / ln(convexHalfCorner)` and then converted the result to `s`, which I think is clearer.

Reply:
Done

Reply:
Done